samples/Intro/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Polly;
+using Polly.Retry;
+using Polly.Timeout;
+
+// ------------------------------------------------------------------------
+// 1. Create a simple resilience strategy using ResilienceStrategyBuilder
+// ------------------------------------------------------------------------
+
+// The ResilienceStrategyBuilder creates a ResilienceStrategy
+// that can be executed synchronously or asynchronously
+// and for both void and result-returning user-callbacks.
+ResilienceStrategy strategy = new ResilienceStrategyBuilder()
+    // Use convenience extension that accepts TimeSpan
+    .AddTimeout(TimeSpan.FromSeconds(5)) 
+    .Build();
+
+// ------------------------------------------------------------------------
+// 2. Execute the strategy
+// ------------------------------------------------------------------------
+
+// synchronously
+strategy.Execute(() => { });
+
+// asynchronously
+await strategy.ExecuteAsync(async token => { await Task.Yield(); }, CancellationToken.None);
```

Comment:
Let's use a short delay instead of yield as I think it would be more readable to a beginner.